cliquet/errors.py

```diff
@@ -176,7 +176,7 @@ def send_alert(request, message=None, url=None, code='soft-eol'):
     :param url: The URL for more information, default to the documentation url.
     """
     if url is None:
-        url = request.registry.settings['cliquet.project_docs']
+        url = request.setting('project_docs')
```

Comment:
I liked it short, but I know what you mean :) This is indeed a broader question : #290